src/pgduckdb_hooks.cpp

```diff
@@ -135,6 +142,27 @@ NeedsDuckdbExecution(Query *query) {
 	return ContainsDuckdbItems((Node *)query, NULL);
 }
 
+/*
+ * We only check ContainsFromClause if duckdb.force_execution is set to true.
+ *
+ * If there's no FROM clause, we're only selecting constants. From a
+ * performance perspective there's not really a point in using DuckDB. If we
+ * forward all of such queries to DuckDB anyway, then many queries that are
+ * used to inspect postgres will throw a warning or return incorrect results.
+ * For example:
+ *
+ *    SELECT current_setting('work_mem');
+ *
+ * So even if a user sets duckdb.force_execution = true, we still won't
+ * forward such queries to DuckDB. With one exception: If the query
+ * requires duckdb e.g. due to a duckdb-only function being used, we'll
+ * still executing this in DuckDB.
+ */
+static bool
+ContainsFromClause(Query *query, bool throw_error = false) {
```

Comment:
Do we really want to keep `throw_error`?

Reply:
nope, copy paste mistake. Fixed now.